docs/sandbox-permissions.rst

```diff
@@ -223,9 +223,82 @@ requested with ``--filesystem`` and are not available with
 
 Device access
 `````````````
+You can provide the following device permissions:
+
+========= ======================================================
+``dri``   Direct Rendering Interface. Necessary for GL.
+``kvm``   Kernel based Virtual Machine ``/dev/kvm``
+``shm``   Shared Memory in ``/dev/shm``.
+``input`` Input devices as exposed in ``/dev/input``. This includes game controllers. Since flatpak 1.15.6.
+``all``   All devices, including all of the above except ``shm``
+========= ======================================================
 
 While not ideal, ``--device=all`` can be used to access devices like
-controllers or webcams.
+generic USB or webcams.
+
+The ``input`` device permission was introduced in 1.15.6. Adding this
+permission will have no effect on prior flatpak versions, and therefor
+prevent using the application as intended. Expecting user to update
+flatpak to the latest version is not possible as some LTS distribution
+will never update. We can expect took take 5 years for most users to
+be on the correct version and by that time, there will be more
+additions as more granular device permission get added.
```

Comment:
I'd just shorten this to a single line: Note that not all distributions or users might not be running latest version of flatpak supporting this feature.

Reply:
Rephrased.